src/components/maincontroller.ts

```diff
@@ -34,16 +32,14 @@ import { MediaSourceInfo } from '~/api/generated/models/media-source-info';
 import { GlobalScope, PlayRequest } from '~/types/global';
 
 window.castReceiverContext = cast.framework.CastReceiverContext.getInstance();
-window.mediaManager = window.castReceiverContext.getPlayerManager();
```

Comment:
rationale here is that in CAF it was renamed to PlayerManager, pre-caf it was called MediaManager and they roughly correspond. Using the new name is more intuitive because you don't need a history book to understand it.